app/assets/stylesheets/chart-styles.css.scss

```diff
@@ -62,4 +62,20 @@
     stroke: black;
   }
 
+  /*JESSIE EDITS*/
+g.data-points circle {
+  opacity: 0.3;
+  fill: grey;}
```

Comment:
Color `grey` should be written in hexadecimal form as `#808080`<br>Color literals like `grey` should only be used in variable declarations; they should be referred to via variable everywhere else.